admin_guide/cluster_metrics.adoc

```diff
@@ -194,7 +197,13 @@ $ oadm policy add-cluster-role-to-user \
     system:serviceaccount:default:heapster
 ----
 
-The following is the definition of the *Heapster* replication controller:
+If you are using heapster for
+link:../dev_guide/pod_autoscaling.html[horizontal pod autoscaling], you'll
+need to use `system:serviceaccount:openshift-infra:heapster` instead of
+`system:serviceaccount:default:heapster`.
```

Comment:
Probably, unless you want to have multiple heapsters running, or something like that.